examples/tensorrt_llm/common/protocol.py

```diff
@@ -38,13 +38,15 @@
 class DynamoTRTLLMCompletionRequest(CompletionRequest):
     id: str = Field(default_factory=lambda: f"cmpl-{str(uuid.uuid4().hex)}")
     max_completion_tokens: Optional[int] = None
+    nvext: Optional[dict] = Field(default=None)
 
 
 class DynamoTRTLLMChatCompletionRequest(ChatCompletionRequest):
     id: str = Field(default_factory=lambda: f"chatcmpl-{str(uuid.uuid4().hex)}")
     max_completion_tokens: Optional[int] = None
     max_tokens: Optional[int] = None
     disaggregated_params: Optional[DisaggregatedParams] = Field(default=None)
+    nvext: Optional[dict] = Field(default=None)
```

Comment:
I think the rust frontend puts `min_tokens` inside `nvext` -- if `nvext` wasn't here before - maybe `min_tokens` isn't getting set in the internal chat request fields for trtllm, and this is why I saw such large variance in num output tokens from genai-perf?

Reply:
This can indeed be the problem. Let's fix this in another PR.